api/main.go

```diff
@@ -58,10 +58,16 @@ func main() {
 		staticCreds := credentials.NewStaticCredentials(cfg.AccessKey(), cfg.SecretKey(), "")
 		awsConfig.WithCredentials(staticCreds)
 		awsConfig.WithRegion(cfg.Region())
-		awsConfig.WithMaxRetries(MAX_AWS_RETRIES)
+		awsConfig.WithMaxRetries(cfg.MaxRetries())
```

Comment:
does this need a value? why shouldn't it be a  super high `const`  like 999?

Reply:
The flag [defaults to 15](https://github.com/quintilesims/layer0/blob/423-retry-logic-sdk-jparsons/common/config/api.go#L92), and simultaneously allows the user to define a larger number if so desired.

Reply:
I worry that's not enough, users would rather see a timeout before a throttled error. I think something like 50 should be the default.

Reply:
A secondary follow-up question: Is it acceptable that this is a configurable value as I've made it to be?

Reply:
yeah i think it can't hurt to have it configurable.